pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -56,10 +56,12 @@
 
     from ._serialize import _serialize_obs, _serialize_ops
     from ctypes.util import find_library
+    from importlib import util as imp_util
 
-    if find_library("custatevec") == None:
+    if find_library("custatevec") == None and not imp_util.find_spec("cuquantum"):
         raise ImportError(
-            'cuQuantum libraries not found. Please check "LD_LIBRARY_PATH" environment variable.'
+            'cuQuantum libraries not found. Please check your "LD_LIBRARY_PATH" environment variable,'
+            'or ensure you have installed the appropriate distributable "cuQuantum" package.'
         )
```

Comment:
Hi @mlxd @maliasadi Thanks for the nice support! I am from the cuQuantum team. @ahehn-nv and I were testing the PennyLane-cuQuantum integration and we noticed this issue (exposed by a silly mistake I made). This `ImportError` is later swallowed at line 72 as a warning, so this error message is gone and a user would only see this:
```
/home/leof/miniforge3/envs/pnl/lib/python3.9/site-packages/pennylane_lightning_gpu/lightning_gpu.py:72: UserWarning: libcustatevec.so.1: cannot open shared object file: No such file or directory
  warn(str(e), UserWarning)
```
I think this is not the intention, right? I confirmed that once we do `pip install cuquantum` and rerun the same code, everything would work (=no more warning). So, I think once fixed, this message should hint users to do `pip install cuquantum`. It's the easiest solution for Python users.

Reply:
Hi @leofang thanks for jumping in. We decided to capture this error as a warning to allow direct fall-back to the CPU-only functionality of `lightning.qubit`, the parent class of `lightning.gpu`. If a user does not have the correct supported hardware or software, we would like them to not be stopped from working. This is more of a quality of life improvement. Though, I do agree we should make it more explicit.

Reply:
@mlxd Right, to the very least the error message should be re-emitted so that users are informed they're not using GPUs and how they can fix it. We've had users (of other frameworks, not Pennylane) reporting the GPU/cuQuantum is not in use despite they have the environment set up, but there's little we can do other than passing the reports to our partners to fix...🙂

Reply:
Yep, totally agree. We will make sure it is prominently displayed in the next release. Thanks again for the feedback.